sdjournal/journal.go

```diff
@@ -220,6 +220,22 @@ func (j *Journal) GetData(field string) (string, error) {
 	return msg, nil
 }
 
+// SetDataThresold sets the data field size threshold for data returned by
+// GetData. To retrieve the complete data fields this threshold should be
+// turned off by setting it to 0, so that the library always returns the
+// complete data objects.
+func (j *Journal) SetDataThreshold(threshold uint64) error {
+	j.mu.Lock()
+	r := C.sd_journal_set_data_threshold(j.cjournal, C.size_t(threshold))
+	j.mu.Unlock()
+
+	if r != 0 {
```

Comment:
Not quite true, we're a bit inconsistent: https://github.com/coreos/go-systemd/pull/124/files#diff-167dc1dd14a090cd485c7dfbb2cfa9b3R262 , https://github.com/coreos/go-systemd/pull/124/files#diff-167dc1dd14a090cd485c7dfbb2cfa9b3R276